easybuild/easyconfigs/f/FastQC/FastQC-0.10.1-Java-1.7.0_80.eb

```diff
@@ -17,22 +17,31 @@ description = """FastQC is a quality control application for high throughput seq
  application to review the results of several different QC checks, or create an HTML based
  report which can be integrated into a pipeline."""
 
-toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchain = {'name': 'dummy', 'version': ''}
 
 sources = ['%(namelower)s_v%(version)s.zip']
 source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
 
+patches = ['FastQC_shebang.patch']
+
 java = 'Java'
 javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
 
-dependencies = [('Java', '1.7.0_80', '', True)]
-
-install_cmd = "cp -a FastQC/* %(installdir)s && chmod +x %(installdir)s/fastqc"
+postinstallcmds = ['chmod +x %(installdir)s/fastqc']
 
 sanity_check_paths = {
+<<<<<<< HEAD:easybuild/easyconfigs/f/FastQC/FastQC-0.10.1-Java-1.7.0_80.eb
+    'files': ['fastqc'],
+    'dirs': [],
+=======
     'files': ["fastqc", "fastqc_icon.ico", "INSTALL.txt", "jbzip2-0.9.jar", "LICENSE.txt",
               "README.txt", "RELEASE_NOTES.txt", "run_fastqc.bat", "sam-1.32.jar"],
     'dirs': ["Contaminants", "Help", "Templates", "uk"],
+>>>>>>> develop:easybuild/easyconfigs/f/FastQC/FastQC-0.10.1-goolf-1.4.10.eb
```

Comment:
merge gone wrong...

can you look into fixing this @nathanhaigh?

may be my bad, let me know if you need help with this